app/src/components/ResourceInfoCard.tsx

```diff
@@ -106,19 +108,31 @@ export default function ResourceInfoCard({ resourceInfo, authoring }: ResourceIn
           <Group>
             <Link
               href={
-                authoring
+                authoring === true
                   ? `/authoring/${resourceInfo.resourceType}/${resourceInfo.id}`
                   : `/${resourceInfo.resourceType}/${resourceInfo.id}`
               }
               key={resourceInfo.id}
             >
-              <Tooltip label={authoring ? 'Edit Draft Resource' : 'View Resource Contents'} openDelay={1000}>
+              <Tooltip label={authoring === true ? 'Edit Draft Resource' : 'View Resource Contents'} openDelay={1000}>
                 <ActionIcon radius="md" size="md" variant="subtle" color="gray">
-                  {authoring ? <Edit size="24" /> : <SquareArrowRight size="24" />}
+                  {authoring === true ? <Edit size="24" /> : <SquareArrowRight size="24" />}
+                </ActionIcon>
+              </Tooltip>
+            </Link>
+            <Link
+              href={{
+                pathname: `/review/${resourceInfo.resourceType}/${resourceInfo.id}`,
+                query: { authoring: `${authoring}` }
+              }}
+            >
+              <Tooltip label={authoring === true ? 'Review Draft Resource' : 'Review Resource'} openDelay={1000}>
+                <ActionIcon radius="md" size="md" variant="subtle" color="#339AF0">
```

Comment:
```suggestion
                <ActionIcon radius="md" size="md" variant="subtle" color="blue">
```
I would just change this to blue- we try to use the Mantine presents whenever possible to make things simpler.